packages/core/src/external/aws/app-config.ts

```diff
@@ -1,63 +1,81 @@
-import { AppConfig } from "aws-sdk";
+import AWS, { AppConfig } from "aws-sdk";
+import { z } from "zod";
 import { MetriportError } from "../../util/error/metriport-error";
 import { out } from "../../util/log";
+import { uuidv4 } from "../../util/uuid-v7";
 
 const { log } = out(`Core appConfig - FF`);
 
+const clientId = uuidv4();
```

Comment:
Any ID that uniquely identifies the caller.

From [the docs](https://docs.aws.amazon.com/appconfig/2019-10-09/APIReference/API_GetConfiguration.html#API_GetConfiguration_RequestSyntax):
> The clientId parameter in the following command is a unique, user-specified ID to identify the client for the configuration.